docker/common.yml

```diff
@@ -59,6 +59,18 @@ superdesk:
    - SUPERDESK_WS_URL=ws://127.0.0.1/ws
    - SUPERDESK_CLIENT_URL=http://127.0.0.1
    - SUPERDESK_TESTING=True
+   - EMBED_PROTOCOL=http://
+   - LIVEBLOG_DEBUG=False
+   - SYNDICATION=True
+   - MONGO_DBNAME=superdesk
+   - SUBSCRIPTION_LEVEL=network
+   - LIVEBLOG_CACHE_TYPE=redis
+   - LIVEBLOG_CACHE_REDIS_URL=redis://redis:6379/12
```

Comment:
Let's remove the redis cache settings, by default in local env the `simple` cache is easier approach.